slack_sdk/socket_mode/websockets/__init__.py

```diff
@@ -56,6 +56,7 @@ class SocketModeClient(AsyncBaseSocketModeClient):
     auto_reconnect_enabled: bool
     default_auto_reconnect_enabled: bool
     closed: bool
+    connect_operation_lock: Lock
```

Comment:
These are the only changes that are necessary in other client implementations. I'll have a migration guide section in the next version's release notes.